osi_sensorview.proto

```diff
@@ -116,12 +196,12 @@ message RadarSensorView
 // \brief Lidar Sensor View
 //
 // Lidar-Specific Sensor View Data
-message LidarSensorView 
+message LidarSensorView
 {
-    // Lidar Input Configuration valid at the time the data was created
+    // Lidar View Configuration valid at the time the data was created
     //
-    optional LidarSensorInputConfiguration lidar_sensor_input_configuration = 1;
-    
+    optional LidarSensorViewConfiguration lidar_sensor_view_configuration = 1;

```

Comment:
The field names of this message are:
- lidar_sensor_view_configuration
- reflections

Is it more clear, if we rename it, isn't it?
- configuration
- reflections

or (but not better)

- lidar_sensor_view_configuration
- lidar_sensor_view_reflections

This also applies to the other XXXSensorView messages.

Reply:
I think shortening to just `configuration` is too short, since it is not just any configuration but the configuration of this view (there might be other configurations, e.g. the configuration of detected objects, ...). The reflections OTOH are not related to the view (they are not reflections of the view but `reflections` of the outside world).

So if one wanted to avoid the mouthful `lidar_sensor_view_configuration`, I'd propose shortening to `view_configuration`, even at the risk of some ambiguity, since there is also the more general SensorViewConfiguration.